tests/test-all-commands.sh

```diff
@@ -47,7 +47,16 @@ npm t
 npm run genprivkey
 cd ..
 
+echo "testing create project --asm hello-world-asm"
+rm -rf hello-world-asm
+node ../src/bin/index.js project --asm hello-world-asm
+cd hello-world-asm
```

Comment:
Edit `.asm/asm.json` before running test, otherwise asm feature is not used and covered.